packages/orm/src/relations/RecursiveCollection.ts

```diff
@@ -250,6 +253,7 @@ export class RecursiveChildrenCollectionImpl<T extends Entity, U extends Entity>
   }
 
   get isLoaded(): boolean {
+    // We could cache this, and add staleness tracking
```

Comment:
Curious: Are these `we could cache this` comments in other files all going to be a follow up?

Reply:
Most will be follow-ups yes, but I went ahead and did the two in this Recursive parent/child.